R/pkg/R/generics.R

```diff
@@ -1126,6 +1126,10 @@ setGeneric("sort_array", function(x, asc = TRUE) { standardGeneric("sort_array")
 #' @export
 setGeneric("soundex", function(x) { standardGeneric("soundex") })
 
+#' @rdname spark_partition_id
+#' @export
+setGeneric("spark_partition_id", function(x) { standardGeneric("spark_partition_id") })
+
```

Comment:
Do you mean before `sd`?
Currently, it is sorted already, isn't it?
`soundex` -> `spark_partition_id` -> `stddev`?